frontend/src/less/main.less

```
@@ -300,7 +300,9 @@ each(@directions, .(@direction) {
     grid-template-columns: @defaultLeftBarWidth 1fr;
   }
 }
-
+.align-self-start {
+  align-self: flex-start;
```

```suggestion
  align-self: start;
```
to align with the one below?